src/helm/benchmark/scenarios/lextreme_scenario.py

```diff
@@ -0,0 +1,405 @@
+import ast
+import random
+from pathlib import Path
+from typing import List
+
+import datasets
+from datasets import load_dataset
+
+from .scenario import Scenario, Instance, Reference, CORRECT_TAG, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT
+
+# SLTC: Single Class Text Classification
+# MLTC: Multi Class Text Classification
+# NER: Named Entity Recognition
+TASK_CODE_MAPPING = {
+    'brazilian_court_decisions_judgment': 'SLTC',
```

Comment:
Could you make "brazilian_court_decisions_judgment", "brazilian_court_decisions_unanimity" etc. a `StrEnum`:

```
from enum import StrEnum

class SubsetName(StrEnum):
    BRAZILIAN_COURT_DECISIONS_JUDGEMENT = auto()
    BRAZILIAN_COURT_DECISIONS_UNANIMITY = auto()
    ...
```

This helps avoid typos.

Likewise for task types:

```
class TaskType(StrEnum):
    SINGLE_CLASS_TEXT_CLASSIFICATION = "SLTC"
    MULTI_CLASS_TEXT_CLASSIFICATION = "MLTC"
    ...
```

Reply:
Good idea, thanks for the suggestion. I implemented it for the task type. Can I use the LowercaseStrEnum (https://pypi.org/project/StrEnum/) for the subset name? Or should I just make it lowercase (needed for the huggingface load_dataset)?

Reply:
did it with constants